CMP1903M Assessment 1 Base Code/Report.cs

```diff
@@ -1,16 +1,47 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace CMP1903M_Assessment_1_Base_Code
-{
-    class Report
-    {
-        //Handles the reporting of the analysis
-        //Maybe have different methods for different formats of output?
-        //eg.   public void outputConsole(List<int>)
-
-    }
-}
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CMP1903M_Assessment_1_Base_Code
+{
+    class Report
+    {
+        //Handles the reporting of the analysis
+        //Maybe have different methods for different formats of output?
+        //eg.   public void outputConsole(List<int>)
+        public static void outputConsole(int sentenceCount, int charCount, int vowelCount,int consCount, int upperCount, int lowerCount, int numCount, int symbCount, int spaceCount,int totalCharCount)//List<int>

```

Comment:
A lot of parameters here. Consider using a single List or Dictionary here to help with polymorphism